bats/tests/containers/factory-reset.bats

```diff
@@ -156,22 +156,20 @@ rdctl_factory_reset() {
 
 check_directories() {
     # Check if all expected directories are created after starting application/ are deleted after a factory reset
-    delete_dir=("$PATH_CONFIG")
+    delete_dir=("$PATH_CONFIG_FILE" "$PATH_LOGS" "$PATH_APP_HOME/credential-server.json" "$PATH_APP_HOME/rd-engine.json")
     if is_unix; then
-        delete_dir+=("$HOME/.rd")
-        delete_dir+=("$PATH_APP_HOME/credential-server.json" "$PATH_APP_HOME/rd-engine.json" "$LIMA_HOME" "$PATH_LOGS")
+        delete_dir+=("$HOME/.rd" "$LIMA_HOME")
```

Comment:
Shouldn't we still include `$PATH_CONFIG` here?

I understand that we don't delete it on Windows because it is the same directory as `$PATH_APP_HOME`, which may include snapshot data.

But on macOS and Linux this is still an empty directory that should be removed, no?

Reply:
Good point. I left it out because on Windows it points to the whole appdata rd directory, which could include snapshots. But on macos and linux it doesn't